Core/Core/Database/CoreData/Models/Assignment.swift

```diff
@@ -131,6 +131,11 @@ extension Assignment {
             !submissionTypes.contains(.none) && !submissionTypes.contains(.on_paper)
     }
 
+    public var isLTIAssignment: Bool {
+        return submissionTypes.count == 1 &&
+            submissionTypes.contains(.external_tool)
```

Comment:
I can't image `.external_tool` being used along with another type, since the web UI doesn't allow that. It's probably still good to have the `count` check, though.

Reply:
Yeah I wasn't sure if the api for whatever reason would allow it even though it not making much sense.